modules/distr-tracing-tempo-config-query-frontend.adoc

```diff
@@ -8,10 +8,10 @@
 
 Two components of the {TempoShortName}, the querier and query frontend, manage queries. You can configure both of these components.
 
-The querier component finds the requested trace ID in the ingesters or back-end storage. Depending on the set parameters, the querier component can query both the ingesters and pull bloom or indexes from the back end to search blocks in object storage. The querier component exposes an HTTP endpoint at `GET /querier/api/traces/<traceID>`, but it is not expected to be used directly. Queries must be sent to the query frontend.
+The querier component finds the requested trace ID in the ingesters or back-end storage. Depending on the set parameters, the querier component can query both the ingesters and pull bloom or indexes from the back end to search blocks in object storage. The querier component exposes an HTTP endpoint at `GET /querier/api/traces/<trace_id>`, but it is not expected to be used directly. Queries must be sent to the query frontend.
 
 .Configuration parameters for the querier component
-[options="header"]
+[options="header",cols="l, a, a]
 |===
```

Comment:
For future ref; I would keep smaller content changes like this in a separate PR from major structural changes if possible, since it makes it easier for folks who are reviewing 🙂

Reply:
🙂 Yes, I completely understand. I try to be reviewer-friendly as much as possible.